es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = 'Irvine Center Drive';
+    const address = `${number} ${street}`;
+
+    console.log(`LearningFuze: ${address}`);
+
+    const firstName = 'Uzair';
+    const lastName = 'Ashraf';
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log(`Bio: ${bio}`);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = fruit + item;
```

Comment:
Missed a concatenation.

Reply:
Practically, is there a reason to use template literals for something like this where there are only variables being concatenated, not strings?

Reply:
Practically, the code may be more or less readable, depending on what it's for. However, this exercise asks you to convert _all_ concatenations to template literals for the practice, so please do so.